signingscript/docker.d/init_worker.sh

```diff
@@ -38,7 +38,7 @@ if [ "$ENV" == "prod" ]; then
   export AUTHENTICODE_TIMESTAMP_STYLE=old
   export AUTHENTICODE_CERT_PATH=/app/signingscript/src/signingscript/data/authenticode_prod_202005.crt
   export AUTHENTICODE_CERT_PATH_202005=/app/signingscript/src/signingscript/data/authenticode_prod_202005.crt
-  export AUTHENTICODE_CA_PATH=/app/signingscript/src/signingscript/data/authenticode_prod_ca_202005.crt
+  export AUTHENTICODE_CA_PATH=/app/signingscript/src/signingscript/data/authenticode_prod_202005.crt
```

Comment:
I think this may work, but do we sign anything with the non-202005 cert?

Reply:
(If so, we may either want to concatenate the crt files into one, or point at the CERT_PATH that we're using. Also, I'm not sure if the current CA_PATH has additional info that may be needed in the .crt. But this is worth a shot.)